dev-docs/bidders/dxtech.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bidder
+title: DXTech
+description: Prebid DXTech Bidder Adapter
+pbjs: true
+pbs: true
+biddercode: dxtech
+media_types: banner, video
```

Comment:
Please add as much meta information as possible. See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter for a full list.

Especially

```
safeframes_ok: true/false
deals_supported: true/false
floors_supported: true/false
fpd_supported: true/false
prebid_member: true/false
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
ortb_blocking_supported: true/partial/false
```

help publishers vet your adapter